plugins/@grouparoo/postgres/__tests__/table-import/import-properties.ts

```diff
@@ -1166,6 +1166,9 @@ describe("postgres/table/recordProperties", () => {
   });
 });
 
-function fixedLengthFloat(value: any, decimalDigits = 2) {
+function fixedLengthFloat(
+  value: Property[] | RecordPropertiesPluginMethodResponse,
+  decimalDigits = 2
+) {
   return parseFloat(parseFloat(value.toString()).toFixed(decimalDigits));
 }
```

Comment:
I wonder if this common method should be shared in `@grouparoo/app-templates` or `@grouparoo/spec-helper` too